doc/rst/technotes/initializers.rst

```diff
@@ -0,0 +1,134 @@
+.. _readme-initializers:
+
+============
+Initializers
+============
+
+The Chapel team is implementing a new approach to user-defined
+intialization of variables with record type or instances of class
+type.  This approach relies on methods known as initializers rather
+than the original methods known as constructors.
+
+A discussion of the current design and rationale is provided in
+`CHIP 10 <https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/10.rst>`_.
+
+Release Chapel 1.15.0 provides a strong preview implementation of this
+new feature. This initial implementation is not of production quality
+and has not been heavily tested, but the basic features are working
+and it is believed that adventurous developers will have success using
+init methods in new code.
+
+It is anticipated that the implementation will continue to advance
+steadily after this release and that many aspects of the internal
+implementation of Chapel will transition from constructor methods to
+initializer methods during this release.
+
+
+Stability
+---------
+
+The functionality of initializer method has four major aspects
```

Comment:
I think you're missing an article here, e.g. "The functionailty of *the* initializer method ..."